Register Grafana in component governance using cgmanifest.json

lpatalas opened the pull request:
We need to register Grafana in component governance for SDL compliance.  It's not picked up by default by automated scanner so we have to add it manually using cgmanifest.json: https://docs.opensource.microsoft.com/tools/cg/cgmanifest.html

Related issue: https://github.com/dotnet/core-eng/issues/11553

riarenas commented:
If we have to do it we have to do it. How do we plan to keep this up to date if we decide to update the grafana version  we use?

Wondering if it makes any sense to generate this file during the build that would take advantage of the grafana deployment script to know which version and url to use? 

Maybe overkill?

garath commented:
Note that the staging instance of Grafana is a different version than prod: 6.5.1. This isn't on purpose, it's just how deployments landed.

lpatalas commented:
@garath I implemented option 2 from the above comment. Additionally I fixed a few issues reported by shellcheck in `setup.sh` file. Let me know what do you think about this?
